templates/components/_nav_mobile.html.twig

```diff
@@ -1,22 +1,28 @@
 <nav class="gtm__navigation__mobile l__wrapper l__hide--above-small-laptop space--20-0">
     <ul class="list__links list__links--row list__links--nav l__row--h-stretch">
         <li><a href="{{ path('homepage') }}" class="link--no-decor text--logo">En Marche !</a></li>
+        <li><a class="btn btn--pink" href="{{ path('donation_index') }}">Donner</a></li>
```

Comment:
This button will then appear twice? :thinking: 
see line 22

Reply:
got it. one in the header & one in the menu

Reply:
Yes, it's normal : )